src/main/java/org/opensearch/plugin/insights/core/service/categorizer/QueryShapeGenerator.java

```diff
@@ -38,13 +38,22 @@ public class QueryShapeGenerator implements ClusterStateListener {
     static final String EMPTY_STRING = "";
     static final String ONE_SPACE_INDENT = " ";
     private final ClusterService clusterService;
-    private final String NO_FIELD_TYPE_VALUE = "";
     private final IndicesFieldTypeCache indicesFieldTypeCache;
+    private long cacheHitCount;
+    private long cacheMissCount;
+
+    private final String NO_FIELD_TYPE_VALUE = "";
+    public static final String HITS_STRING = "hits";
+    public static final String MISSES_STRING = "misses";
+    public static final String COUNT_STRING = "count";
+    public static final String BYTES_STRING = "bytes";
```

Comment:
Can we rename these to below for better clarity and consistency? And, also add eviction metric as well!?

```
"size_in_bytes": 4096,
"entry_count": 1000,
"evictions": 2,
"hit_count": 60,
"miss_count": 10
```

Reply:
Done